tests/introspection/Mac/MacApiSelectorTest.cs

```diff
@@ -381,7 +381,12 @@ protected override bool Skip (Type type, string selectorName)
 						return true;
 					}
 					break;
-
+				case "NSMenuView":
+					switch (selectorName) {
```

Comment:
Why switch here and not just an if?

Reply:
easier to add more items later, with a minimal diff